shell/edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -2059,7 +2078,18 @@ export default {
         description-label="cluster.description.label"
         description-placeholder="cluster.description.placeholder"
         :rules="{ name: fvGetAndReportPathRules('metadata.name') }"
-      />
+      >
+        <template
+          v-if="showClusterAppearance"
```

Comment:
the component `ClusterAppearance ` seems to be ready to work with both `EDIT` and `CREATE` scenarios, but here we are gating it only to a `CREATE` scenario... Is this intended?